azext_iot/central/_help.py

```diff
@@ -316,6 +316,20 @@ def _load_central_monitors_help():
             az iot central app validate-messages --app-id {app_id} -d {device_id} --cg {consumer_group_name}
     """
 
+    helps[
+        "iot central app monitor-properties"
+    ] = """
+        type: command
+        short-summary: Monitor desired and reported  sent to/from the IoT Hub for an IoT Central app.
```

Comment:
nit: double space between "reported" and "sent"
related nit: Monitor desired and reported what? Looks like a word (properties?) is missing.